### PR TITLE
Json missing targetdevice parameter

### PR DESCRIPTION
**Pull request by JoeOster**
# Existing Sample Changes
## Description

Found some json missing data, specifically the TargetDevice parameter

### CODE REVIEW

**Comment by louie-tsai** [comment]
looks good to me for AI samples.

**Comment by DmitryDenisenko751** [comment]
With this change, DirectProgramming/DPC++FPGA/ReferenceDesigns/qrd/sample.json has targetDevice appear twice. Looks like this file does not need to be changed at all.

File: DirectProgramming/DPC++FPGA/ReferenceDesigns/qrd/sample.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
   "builder": ["ide", "cmake"],
   "targetDevice": ["FPGA"],
   "languages": [{"cpp":{}}],
+  "targetDevice": [ "FPGA" ],
```
Comment (mdbtucker) [inline DirectProgramming/DPC++FPGA/ReferenceDesigns/qrd/sample.json:11]: This file already contains a target device (look at line 9).  Please remove this file from this PR.

File: DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/compute_units/sample.json [hunk 2]
```diff
@@ -5,9 +5,9 @@
   "description": "Intel® FPGA tutorial showcasing a design pattern to enable the creation of compute units",
   "toolchain": ["dpcpp"],
   "os": ["linux", "windows"],
-  "targetDevice": ["FPGA"],
   "builder": ["ide", "cmake"],
   "languages": [{"cpp":{}}],
+  "targetDevice": [ "FPGA" ],
```
Comment (mdbtucker) [inline DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/compute_units/sample.json:10]: I don't understand why this line needs to be moved?

File: AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_InferenceOptimization/sample.json [hunk 3]
```diff
@@ -8,6 +8,7 @@
 		"python": {}
 	}],
 	"os": ["linux"],
+        "targetDevice": ["CPU"],
```
Comment (mdbtucker) [inline AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_InferenceOptimization/sample.json:11]: Indent looks wrong here?

File: AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8/sample.json [hunk 4]
```diff
@@ -8,6 +8,8 @@
 		"python": {}
 	}],
 	"os": ["linux"],
+        "targetDevice": ["CPU"],
```
Comment (mdbtucker) [inline AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8/sample.json:11]: Indent problem?

**Review by mdbtucker — DISMISSED** [review]
Looks good other than the indent issues noted in my comments, so approving.

File: AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_InferenceOptimization/sample.json (hunk 3 above)
Comment (JoeOster) [reply AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_InferenceOptimization/sample.json]: fixed

File: AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8/sample.json (hunk 4 above)
Comment (JoeOster) [reply AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8/sample.json]: fixed

File: DirectProgramming/DPC++FPGA/ReferenceDesigns/qrd/sample.json (hunk 1 above)
Comment (JoeOster) [reply DirectProgramming/DPC++FPGA/ReferenceDesigns/qrd/sample.json]: fixed

File: DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/compute_units/sample.json (hunk 2 above)
Comment (JoeOster) [reply DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/compute_units/sample.json]: fixed

**Review by mdbtucker — APPROVED** [review]

**Pull request merged**